tests/PagesAreFoundTest.php

```diff
@@ -2,21 +2,53 @@
 
 namespace App\Tests;
 
+//use App\Repository\UserRepository;
```

Comment:
Waarom in comment?

Reply:
Dit is enkel nodig indien er moet ingelogd worden voor een bepaalde pagina. Alles wat in comment staat in deze test dient daarvoor.

Reply:
Misschien dan in comment erbij zetten

Reply:
@tijsverkoyen comment is toegevoegd 359ee131da4daddddfee2c27efb8b6896f9aa50d